man/org.freedesktop.sysupdate1.xml

```diff
@@ -274,6 +286,38 @@ node /org/freedesktop/sysupdate1/target/host {
       <varname>IMAGE_VERSION</varname> in <filename>/etc/os-release</filename>. If the target has no current
       version, the function will return an empty string.</para>
 
+      <para><function>ListFeatures()</function> returns a list of this target's optional features, by ID.
+      If the target has no optional features, the method returns an empty array.</para>
+
+      <para><function>DescribeFeature()</function> returns all known information about a given optional
```

Comment:
Should use flags

Reply:
Apparently in the code I read a variable that wasn't defined in the dbus. So no, shouldn't use flags. There are no flags to pass.